ocf_data_sampler/select/select_spatial_slice.py

```diff
@@ -97,8 +93,8 @@ def _get_idx_of_pixel_closest_to_poi(
     x_index = da.get_index(x_dim)
     y_index = da.get_index(y_dim)
 
-    closest_x = x_index.get_indexer([x], method="nearest")[0]
-    closest_y = y_index.get_indexer([y], method="nearest")[0]
+    closest_x = float(x_index.get_indexer([x], method="nearest")[0])
+    closest_y = float(y_index.get_indexer([y], method="nearest")[0])
```

Comment:
why is this needed now?

Reply:
@peterdudfield is this required?

Reply:
i got some pydnatic error before, but I can try removing it again

Reply:
i removed it